src/types/user.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  EmailString,
+  FiscalCode,
+  NonEmptyString
+} from "@pagopa/ts-commons/lib/strings";
+import * as t from "io-ts";
+
+export const SpidUser = t.intersection([
+  t.interface({
+    // the following values may be set
+    // by the calling application:
+    // authnContextClassRef: SpidLevel,
+    // issuer: Issuer
+    fiscalNumber: FiscalCode,
```

Comment:
If we use this typing for `fiscalNumber` we need to remap the fiscal code before the `SpidUser.decode` like into the `validateSpidUser` method in [io-backend](https://github.com/pagopa/io-backend/blob/ed049604437adeebca31d47f6a8432c2042c1e59/src/types/user.ts#L187) otherwise the decode always fail.

Reply:
Yep you're right, I will change :)